internal/core/metadata/router.go

```diff
@@ -16,6 +16,7 @@ package metadata
 import (
 	"net/http"
 
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/config"
 	metadataContainer "github.com/edgexfoundry/edgex-go/internal/core/metadata/container"
```

Comment:
For consistency with similar refactorings in other services; this should not be aliased (i.e. it should be internally referenced as `container`)